framework/src/base/Moose.C

```diff
@@ -201,6 +203,8 @@ addActionTypes(Syntax & syntax)
   registerTask("setup_function_complete", false);
   registerTask("setup_variable_complete", false);
   registerTask("ready_to_init", true);
+  appendMooseObjectTask("ready_to_init", InterfaceKernel);
```

Comment:
I'd prefer if these lines were moved up and clustered with existing `ready_to_init` tasks in the section up above.

Reply:
Huh? The only three occurrences of `ready_to_init` are right here.

Reply:
oh weird... I'm not sure if this is a bug or not. Note the name of the method used here. It's called "appendMooseObjectTask", which is supposed to correspond with "registerMooseObjectTask". The first registration or `ready_to_init` is not a MOOSE object task so I hope that the code is doing the right thing. Note that this is definitely a unique combination of these registration functions so it should probably be checked.